modules/nw-network-plugin-migration-process.adoc

```diff
@@ -51,6 +51,11 @@ Cluster:: As nodes reboot, the cluster assigns IP addresses to pods on the {sdn}
 ifeval::["{context}" == "migrate-from-openshift-sdn"]
 If a rollback to OpenShift SDN is required, the following table describes the process.
 
+[IMPORTANT]
+====
+You must wait until the migration process from OpenShift SDN to OVN-Kubernetes network plugin is successful before initiating a rollback.
+====
```

Comment:
Ensure the network and machine-config cluster operators are not in the `progressing` or `degraded` state before starting rollback.

Reply:
@pliurh  I assume the a "successful" migraiton shall indicate that "the network and machine-config cluster operators are not in progressing or degraded state", wdyt?